doc/conf_ref.rst

```diff
@@ -396,6 +396,11 @@ Right side of every repo option can be enriched by the following variables:
 
     Refers to the release version of operating system which DNF derives from information available in RPMDB.
 
+
+In addition to these hard coded variables, user-defined ones can also be used. They can be defined either via :ref:`variable files <varfiles-label>`, or by using special environmental variables. The names of these variables must be prefixed with DNF_VARS\_ and they can only consist of alphanumeric characters and underscores::
+
```

Comment:
Why there is `'\'` character before second underscore in `DNF_VARS\_`?

Reply:
Trailing underscore in restructured text creates a hyperlink, so it has to be escaped here.